index.html

```diff
@@ -152,7 +152,7 @@ <h2 id="docconformance">
         </tr>
         <tr id="ahref" tabindex="-1">
           <td>
-          <a><code>a</code></a> element with a <code>href</code></td>
+          <a><code>a</code></a> element with a <a data-cite="html/links.html#attr-hyperlink-href"><code>href</code></a></td>
```

Comment:
```suggestion
          <a>`a`</a> element with a <a data-cite="html/links.html#attr-hyperlink-href">`href`</a></td>
```

Reply:
@marcoscaceres you seem to be suggesting the removal of the code elements and replacing them with markdown.  The code element is being used in the rest of the document. 

If this is part of a larger change to the markup, I'm not sure if this is the best place to start it.

Reply:
Yeah, we can do those in a new pull request.